src/geff/utils.py

```diff
@@ -28,6 +28,14 @@ def validate(path: str | Path):
 
     graph = zarr.open(path, mode="r")
 
+    # Check if zattrs are empty or missing
+    if not graph.attrs:
```

Comment:
The root of a zarr does have attrs, so I don't think this covers the case you wanted to cover. It's still a great check though! I would end this error message at "This may indicate the path is incorrect", and then add another check right below to see if "geff_version" is in the zattrs. If it is not, we can say the second part about the group name not being right.